examples/image_classification_kornia.ipynb

```diff
@@ -0,0 +1,3856 @@
+{
```

Comment:
<p>Maybe we can add an additional sentence to make it clearer:</p><p> </p><p>All Hugging Face model classes expect the input to be correctly preprocessed. Preprocessing images typically comes down to:</p><ol><li> Resizing ...</li><li>Normalizing...</li><li>...</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/226/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>